L1Trigger/CSCCommonTrigger/src/vlib.cc

```diff
@@ -1,1339 +1,1216 @@
+  mode = mnum;
 #ifdef VGEN
-	ostringstream ln;
-	ln << dec << n;
-	init (Sizeofrval * 8 - 1, 0, ln.str().c_str());
+  ostringstream ln;
+  ln << dec << n;
+  init(Sizeofrval * 8 - 1, 0, ln.str().c_str());
 #else
-	init (Sizeofrval * 8 - 1, 0, "");
+  init(Sizeofrval * 8 - 1, 0, "");
 #endif
-	inited = 0;
-	r = (rval)n;
-	rc = r;
+  inited = 0;
+  r = (rval)n;
+  rc = r;
 }
 
-void Signal::makemask() 
-{
-	mask = (1LL << (h - l + 1)) - 1LL;
-	if (mask == 0LL) mask = ~mask;
+void Signal::makemask() {
+  mask = (1LL << (h - l + 1)) - 1LL;
+  if (mask == 0LL)
+    mask = ~mask;
 }
 
-
-void Signal::init (int high, int low, const char* rname)
-{
-	
+void Signal::init(int high, int low, const char* rname) {
 #ifdef VGEN
-	name = rname;
-	orname = rname;
-	lb = "";
-	rb = "";
+  name = rname;
+  orname = rname;
+  lb = "";
+  rb = "";
 #endif
-	if (!inited)
-	{
-		h = high;
-		l = low;
-		makemask();
-		inited = 1;
-	}
-	source = this;
+  if (!inited) {
+    h = high;
+    l = low;
+    makemask();
+    inited = 1;
+  }
+  source = this;
 }
 
-void Signal::init(Signal* shost, int high, int low, const char* rname)
-{
-	host = shost; 
+void Signal::init(Signal* shost, int high, int low, const char* rname) {
+  host = shost;
 #ifdef VGEN
-	name = rname;
-	orname = rname;
-	lb = "";
-	rb = "";
+  name = rname;
+  orname = rname;
+  lb = "";
+  rb = "";
 #endif
-	h = high;
-	l = low;
-	makemask();
-	source = this;
-	if (host) 
-	{
-		rc = (host->getr() >> (l - host->getl())) & mask;
-		r = rc;
-	}
-	change = pedge = nedge = 0;
+  h = high;
+  l = low;
+  makemask();
+  source = this;
+  if (host) {
+    rc = (host->getr() >> (l - host->getl())) & mask;
+    r = rc;
+  }
+  change = pedge = nedge = 0;
 }
 
 #ifdef VGEN
-string& Signal::getcatname()
-{
-	if (lb[0] == '{')
-		catname = lb + name + rb; 
-	else
-		catname = name;
-	return catname;
+string& Signal::getcatname() {
+  if (lb[0] == '{')
+    catname = lb + name + rb;
+  else
+    catname = name;
+  return catname;
 }
 #endif
 
 // unary operators -------------------------------------
 
 #ifdef VGEN
-#define unop(op,cop) \
-Signal Signal::op () \
-{ \
-	Signal t; \
-	t.name = #cop + lb + name + rb; \
-	printable = 0; \
-	t.r = mask & (cop getval()); \
-	return t; \
-}
+#define unop(op, cop)               \
+  Signal Signal::op() {             \
+    Signal t;                       \
+    t.name = #cop + lb + name + rb; \
+    printable = 0;                  \
+    t.r = mask & (cop getval());    \
+    return t;                       \
+  }
 #else
-#define unop(op,cop) \
-Signal Signal::op () \
-{ \
-	Signal t; \
-	t.init(h, l, ""); \
-	t.r = mask & (cop getval()); \
-	return t; \
-}
+#define unop(op, cop)            \
+  Signal Signal::op() {          \
+    Signal t;                    \
+    t.init(h, l, "");            \
+    t.r = mask & (cop getval()); \
+    return t;                    \
+  }
 #endif
 
-unop (operator!,!)
-unop (operator~,~)
+unop(operator!, !) unop(operator~, ~)
```

Comment:
@smuzaffar I am not sure whether this is really desired, as well as the other similar cases in this file,  although there is no problem observed in the test